### ISSUE TITLE
Offline causes API error message - screenshot attached

### ISSUE THREAD

**Issue by asheroto** [keybase/client#17696]
Keybase GUI Version: 4.0.0-20190507154537+6614a49937

![2019-05-24-24_000067 jpg](https://user-images.githubusercontent.com/49938263/58339604-902d2e00-7e0f-11e9-964e-4e46d2e0b03a.jpg)

**Comment by lokke3** [comment]
I am also experiencing this 
![Screenshot from 2019-06-07 18-14-15](https://user-images.githubusercontent.com/37903981/59136228-29039380-8950-11e9-9f92-57592c912699.png)

> $ keybase log send
> This command will send recent keybase log entries to keybase.io
> for debugging purposes only.
> 
> These logs don’t include your private keys or encrypted data,
> but they will include filenames and other metadata keybase normally
> can’t read, for debugging purposes.
> 
> Continue sending logs to keybase.io? (type 'YES' to confirm): YES
> Enter feedback (or <Enter> to send): Getting ERROR CODE 1601 and a complaint abo
> ut an invalid cert but only on desktop
> More feedback (or press <Enter> when done): 
> ▶ ERROR API network error: Post https://api-0.core.keybaseapi.com/_/api/1.0/logdump/send.json: x509: certificate is valid for *.moprestamo.com, not api-0.core.keybaseapi.com
> 

tried run_keybase didn't help.

> 
> $ run_keybase
> Shutting down Keybase GUI...
> Unmounting /run/user/1000/keybase/kbfs...
> Unmounting and shutting down kbfsfuse...
> Shutting down keybase service...
> Starting via systemd...
> run_keybase: Success!

**Comment by maxtaco** [comment]
Are you behind  a proxy?

**Comment by asheroto** [comment]
I disabled my network connection and that occurred.

**Comment by lokke3** [comment]
I rebooted my router and was able to connect.

**Comment by asheroto** [comment]
Yeah I don't think it's really a bug so much as it need to be a cleaner message such as "No network connection found"
